src/sdk/python/examples/basic/.env

```diff
@@ -1,4 +1,3 @@
 DOCKER_HTTP_PROXY=http://host.docker.internal:3128
```

Comment:
Since the proxies has been removed in general they should be removed here as well.

Reply:
Removed all ocurrances in the environment configurations